tests/backup/backup_helper.go

```diff
@@ -1181,13 +1247,128 @@ func backupSuccessCheck(backupName string, orgID string, retryDuration int, retr
+	for _, clusterAppsContext := range clusterAppsContexts {
+		var specObjects []interface{} = make([]interface{}, 0)
+		// the Backup objects which were found in this namespace
+		var resInfoObjs []*api.ResourceInfo = make([]*api.ResourceInfo, 0)
+
+		for _, resource := range resourceInfos {
+			if resource.Namespace == clusterAppsContext.ScheduleOptions.Namespace {
+				resInfoObjs = append(resInfoObjs, resource)
+			} else if resource.Namespace == "" {
+				// IF it is therein this context, we'll work with it
+				resInfoObjs = append(resInfoObjs, resource)
+			}
+		}
+
+	specloop:
+		for _, spec := range clusterAppsContext.App.SpecList {
+			name, kind, ns, err := GetSpecNameKindNamepace(spec)
+			if err != nil {
+				log.Error(err)
+				continue specloop
+			}
+			if name != "" /*&& kind != ""*/ /*kind is empty for certain objects :(*/ {
+				for _, bkpObj := range resInfoObjs {
+					if name == bkpObj.Name /*&& kind == bkpObj.Kind*/ {
+						clone := spec
+						// The spec for the object/resource was found in the Context (namespace). Appending it.
+						if ns == "" {
+							log.InfoD("The resource(name: %s, kind: %s, with NO NAMESPACE value) was found in the context '%s'", name, kind, clusterAppsContext.ScheduleOptions.Namespace)
+						}
+						specObjects = append(specObjects, clone)
+						continue specloop
+					}
+				}
+			} else {
+				log.Warnf("some error with getting GetSpecNameKindNamepace with Spec Name: %s, Kind: %s, Namespace: %s, in local Context (NS): %s", name, kind, ns, clusterAppsContext.ScheduleOptions.Namespace)
+				continue specloop
+			}
+			// executes if specObj is not in backup
+			log.Warnf("This Backup does not include the resource(name: %s, kind:%s, namespace:%s) found in the local context (NS) '%s'", name, kind, ns, clusterAppsContext.ScheduleOptions.Namespace)
+		}
+
+		// Duplicate the object
+		backupAppContext := *clusterAppsContext
+		// Duplicate the object
+		app := *clusterAppsContext.App
+		app.SpecList = specObjects
+		backupAppContext.App = &app
+		backupAppContexts = append(backupAppContexts, &backupAppContext)
+		allBkpObjs = append(allBkpObjs, resInfoObjs...)
+	}
+
+	// Error Checking: if the contexts for all backed up resources were found in SOME namespace
+
+	var namespaces []string = nil
+	getNS := func() {
+		if namespaces == nil {
+			for _, clusterAppsContext := range clusterAppsContexts {
+				namespaces = append(namespaces, clusterAppsContext.ScheduleOptions.Namespace)
+			}
+		}
+	}
+
+	for _, resInfo := range resourceInfos {
+		found := false
+		for _, bkpObj := range allBkpObjs {
+			if bkpObj.Name == resInfo.Name {
+				// the resInfo object was found in one of the namespaces
+				if resInfo.Namespace == "" {
+					getNS()
+					log.Warnf("The backed up resource: '%s', MAY not have been part of any of the contexts (namespaces); But they WERE checked for in ALL contexts (namespaces): %v. Check previous log entries to verify.", resInfo.Name, namespaces)
```

Comment:
Keep these warning messages short and crisp

Reply:
**I have significantly shortened the error messages and even cleaned and consolidated logging, but I still left in _really short and crisp help messages_. After going through them, if you still feel that it is way too much OR unnecessary OR confusing in nature OR some other reason, I'll restrict it to error-only messages. 😄**

**My reason for error + help messages**: I was treating torpedo as a product, whose 'customers' are the developers/testers using it to check product issues. While error messages tell what went wrong, I felt that, given the complexity of the test case, intimating the 'customer' of possible known solutions or issues might help the developer find a solution more quickly, or at least rule out possibilities, thus increasing productivity by saving time.
The rust compiler's model of `error + extremely detailed help messages` which is extremely popular with devs, is what inspired me to do that.  
Of course, I am also aware of the fact that this can easily clutter up logs and cause confusion due to possibly overwhelming amount of information, as help is combined with error messages. Or this can confuse developers in certain cases, or might not even help save a lot of time due to the complex nature of things.